aspnetcore/signalr/configuration.md

```diff
@@ -162,6 +162,31 @@ let connection = new signalR.HubConnectionBuilder()
     .build();
 ```
 
+::: moniker range=">= aspnetcore-3.0"
+
+Instead of a `LogLevel` value, you can also provide a `string` value representing a log level name. This is useful when configuring SignalR logging in environments where you don't have access to the `LogLevel` constants.
+
+```javascript
+let connection = new signalR.HubConnectionBuilder()
+    .withUrl("/myhub")
+    .configureLogging("warn")
+    .build();
+```
+
+The below table lists the available log levels. The value you provide to `configureLogging` sets the **minimum** log level that will be logged. Messages logged at this level, **or the levels listed below it on the table**, will be logged.
```

Comment:
["the following x" is preferred over the "the below x"](https://worldready.cloudapp.net/Styleguide/Read?id=2700&topicid=27393) because it sounds better when listening to a screen reader.

```suggestion
The following table lists the available log levels. The value you provide to `configureLogging` sets the **minimum** log level that will be logged. Messages logged at this level, **or the levels listed after it in the table**, will be logged.
```